src/nyctaxi/data_processor.py

```
@@ -0,0 +1,56 @@
+
+#from pyspark.sql import SparkSession
+from databricks.connect import DatabricksSession
+import pandas as pd
+import yaml
```

Ruff

`yaml` imported but unused

Remove unused import: `yaml`

(F401)